src/components/BulkMemberManager/BulkMemberManager.js

```diff
@@ -0,0 +1,266 @@
+import i18n from '@dhis2/d2-i18n'
+import { DataTableToolbar, Pagination, SegmentedControl } from '@dhis2/ui'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import styles from './BulkMemberManager.module.css'
+import { useResults } from './hooks/useResults'
+import PendingChanges from './PendingChanges'
+import {
+    addEntity,
+    removeEntity,
+    cancelAddEntity,
+    cancelRemoveEntity,
+} from './pendingChangesActions'
+import PendingChangesPropType from './PendingChangesPropType'
+import ResultsTable from './ResultsTable'
+import TopBar from './TopBar'
+
+const BulkMemberManager = ({
```

Comment:
This component will be extracted from this app and merged into `@dhis2/ui` in future.